suites/octopus/ansible/sanity_containerized_ceph_ansible.yaml

```diff
@@ -0,0 +1,266 @@
+tests:
+   - test:
+       name: install ceph pre-requisites
+       module: install_prereq.py
+       abort-on-fail: true
+
+   - test:
+        name: containerized ceph ansible
+        polarion-id: CEPH-83571503
+        module: test_ansible.py
+        config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: octopus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              ceph_docker_image: rhceph/rhceph-3-rhel7
+              ceph_docker_image_tag: latest
```

Comment:
probably in another pr these can be removed. once rhcs5.x GA can add them backwith proper creds.

everything else LGTM